src/components/AssetsEditor/SingleAssetEditor/SingleAssetEditor.tsx

```diff
@@ -67,8 +66,9 @@ export default class SingleAssetEditor<T extends RawAsset | Asset> extends React
     const { asset } = this.props
 
     let mappings: Asset['contents'] | RawAsset['contents']
+    const contentValues = Object.values(asset.contents)
 
-    if (!isRemoteURL(asset.thumbnail)) {
+    if (typeof contentValues[0] !== 'string') {
```

Comment:
I'm not qualified to attest if this is right or wrong but when I click the checkbox twice I get:

![image](https://user-images.githubusercontent.com/692077/65537080-f53a8280-deda-11e9-90b7-7523e56be524.png)

cc: @cazala

Reply:
oldy file